src/xenia/app/emulator_window.h

```diff
@@ -25,6 +25,15 @@
 #include "xenia/ui/windowed_app_context.h"
 #include "xenia/xbox.h"
 
+enum menuIndex {
```

Comment:
this should be made an `enum class` instead to match with our code style.

something like
```cpp
enum class MenuIndex {
  kFile,
  kCpu
  ...
```